internal/raft/message/convert.go

```diff
@@ -0,0 +1,971 @@
+package message
+
+import (
+	"github.com/tomarrell/lbadd/internal/compiler/command"
+)
+
+// ConvertCommandToMessage converts a command.Command to a message.Command
+func ConvertCommandToMessage(cmd command.Command) Message {
+	switch c := cmd.(type) {
+	case *command.Scan:
+		return ConvertCommandToMessageScan(c)
+	case *command.Select:
+		return ConvertCommandToMessageSelect(c)
+	case *command.Project:
+		return ConvertCommandToMessageProject(c)
+	case *command.Delete:
+		return ConvertCommandToMessageDelete(c)
+	case *command.DropIndex:
+		return ConvertCommandToMessageDrop(c)
+	case *command.DropTable:
+		return ConvertCommandToMessageDrop(c)
+	case *command.DropTrigger:
+		return ConvertCommandToMessageDrop(c)
+	case *command.DropView:
+		return ConvertCommandToMessageDrop(c)
+	case *command.Update:
+		return ConvertCommandToMessageUpdate(c)
+	case *command.Join:
+		return ConvertCommandToMessageJoin(c)
+	case *command.Limit:
+		return ConvertCommandToMessageLimit(c)
+	case *command.Insert:
+		return ConvertCommandToMessageInsert(c)
+	}
+	return nil
+}
+
+// ConvertCommandToMessageTable converts a command.Table to a SimpleTable
+func ConvertCommandToMessageTable(cmd command.Table) *SimpleTable {
+	simpleTable := &SimpleTable{}
+	simpleTable.Schema = cmd.(*command.SimpleTable).Schema
```

Comment:
What if `cmd` is not a simple table? You should probably return an error in that case.

Reply:
Only `SimpleTable` was the implemented struct, so this was assumed. But sure for consistenct will do.

Reply:
Yes right now, but I imagine there will be a lot more than just a simpletable
Also, I don't see any change here (again). GitHub?